packages/vue/src/components/atoms/SfIcon/SfIcon.js

```diff
@@ -29,7 +29,7 @@ export default {
     /**
      * Custom color of the icon
      * It can be according to our standard colors, or legitimate CSS color such as `#fff`, `rgb(255,255,255)`), and `lightgray` or nothing.
-     * Standard colors: `primary`, `secondary`, `white`, `black`, `accent`, `green-primary`, `green-secondary`, `gray-primary`, `gray-secondary`, `light-primary`, `light-secondary`, `pink-primary`, `pink-secondary`, `yellow-primary`, `yellow-secondary`, `blue-primary`, `blue-secondary`.
```

Comment:
@mayashavin imo we should have `primary` color. When you set you brand color probably you want use it.

Reply:
we could also call it "accent"

Reply:
@pspaczek @filrak We should have only one color name instead of using the same color for primary, green-primary and accent. That's why I removed everything and left only `green-primary`. Any better solution?

Reply:
@pspaczek I missed out the typography scss file you mentioned about the $font-size-small-mobile. Will change it accordingly

Reply:
I think we must have `accent` for brand color. We can rename `primary` in variables to `accent` and use it in css for all brand specific palces. 
```
// Primary
/* Should be used to drive attention to the main tasks that should be done while using the app.
 * It is meant to be used in major interactive elements of the page. */
$c-accent: $_c-green-primary !default;
$c-accent-variant: $_c-green-secondary !default;
$c-on-accent: $_c-light-secondary !default;
```

About typogaphy, when we change SCSS to CSS then we have just.
```
$font-size-extra-small: 0.75rem !default;
$font-size-small: 0.875rem !default;
$font-size-regular: 1rem !default;
$font-size-big: 1.125rem !default;
$font-size-extra-big: 1.5rem !default;
```

Reply:
@pspaczek When we rename `primary` to `accent`, there is a need to check all the `primary` used throughout the library components, which is not recommended. Also I don't think `accent` is a good name for branding, especially to replace `primary`. It is a variant of color, but shouldn't be the brand color name itself. @filrak wdyt?
Regarding the typography, I will move your suggestion to the related issue and work on it once this one is merged to avoid further conflict.